patterns/2-structured/maturity-model.md

```diff
@@ -208,6 +208,12 @@ Prior to adopting InnerSource as a working model, teams are aware of the
 practices that are expected of them - both in the short term and in the
 long term.
 
+## Related Patterns
+
+* [Dedicated Community Leader](https://patterns.innersourcecommons.org/p/dedicated-community-leader)
+* [Circle Communities](https://patterns.innersourcecommons.org/p/circle-communities)
+* [Agile InnerSource Dojo](https://patterns.innersourcecommons.org/p/agile-innersource-dojo)
```

Comment:
Let's handle the related patterns in a separate PR.

Reply:
I removed the section.  Not sure if there is a better way to remove the file from the PR overall.

Reply:
The file has disappeared from the PR already, as it did not contain any changes any more. 👍